### PR TITLE
Redistribute fees collector fees

### PR DESCRIPTION
**Pull request by dorin-iancu**

### CODE REVIEW

File: energy-integration/fees-collector/src/redistribute_rewards.rs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        for week in start_week..=end_week {
+            self.accumulate_remaining_rewards_single_week(&mut all_rewards, &all_tokens, week);
+        }
+
+        for reward_entry in &all_rewards {
+            if reward_entry.amount == 0 {
+                continue;
+            }
+
+            self.accumulated_fees(current_week, &reward_entry.token)
```
Comment (psorinionut) [inline energy-integration/fees-collector/src/redistribute_rewards.rs:44]: So we just accumulate the remaining rewards, with no means of claiming them?

Comment (dorin-iancu) [reply energy-integration/fees-collector/src/redistribute_rewards.rs:44]: No, they're put in current week storage, so users can claim them starting next week.

File: energy-integration/fees-collector/src/claim.rs [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        let remaining_rewards_mapper = sc.remaining_rewards(week);
+        let mut remaining_rewards = remaining_rewards_mapper.get();
+        for (i, weekly_reward) in total_rewards.iter().enumerate() {
+            let reward_amount = weekly_reward.amount * energy_amount / total_energy;
+            if reward_amount == 0 {
+                continue;
+            }
+
+            let mut rem_rew_entry = remaining_rewards.get_mut(i);
+            rem_rew_entry.amount -= &reward_amount;
```
[the remark this answers is not in the document]
Comment (dorin-iancu) [reply energy-integration/fees-collector/src/claim.rs:157]: Yes, I'm pretty sure. They're both set at lines 178-179 in this file, and they initially have the exact same tokens and values too.

**Review by psorinionut — APPROVED** [review]
Should not be merged until we have a clear launch date.

**Pull request merged**